tutorial/installation.md

```diff
@@ -163,26 +140,23 @@ If you get an error at step 4 above, or can't launch `napari` after
 installation, you should try to delete your `napari-tutorial` environment, and
```

Comment:
> Sometimes, `napari` installation can fail on an M1 Mac due to mismatching
dependencies on `pip`.

Is this still relevant?

Reply:
yes, I was just running into this problem myself last week

PyQt needs to be installed via conda for things to work on mac m1